src/ui/components/form/form-component/FormComponent.test.tsx

```diff
@@ -39,23 +39,19 @@ const formsTestHandler: FormTestParams = (
 ) => {
   describe(`${testTitle} form`, (): void => {
     it('When submit without filling in fields, it should display errors', () => {
-      const submitHandler = jest.fn((e) => e.preventDefault());
       const { getAllByText, container } = render(
         <FormComponent
-          onSubmit={submitHandler}
```

Comment:
linter was complaining also about these since the component doesn't take onSubmit as param. so deleted for now at least